source/command/atlas-api.txt

```diff
@@ -0,0 +1,176 @@
+.. _atlas-api:
+
+=========
+atlas api
+=========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Access all features of the `{+atlas-admin-api+} <https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/>`__ 
+by using the {+atlas-cli+} with the syntax: ``atlas api <tag> <operationId>``.
+
+This experimental feature streamlines script development by letting you interact 
+directly with any {+atlas-admin-api+} endpoint by using the {+atlas-cli+}.
+
+To learn more about working with the {+atlas-admin-api+}, see :ref:`atlas-admin-api-access`.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas api <tag> <operationId> [options]
+
+.. important::
+   
+   Both ``<tag>`` and ``<operationId>`` must be in :wikipedia:`camelCase <Camel_case>`.
+
+Arguments  
+---------  
+
+.. list-table::
```

Comment:
nit: Can we improve the UX of this table by updating the widths and the spacing in the description?
i.e. expand name and type columns a bit longer so they don't wrap

<img width="792" alt="Screenshot 2025-01-24 at 3 24 03 PM" src="https://github.com/user-attachments/assets/82a1535b-8425-4c13-acce-86f3ff2ef616" />